recipes/summa/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "SUMMA" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "f117aaa889ac785cbde4cd236ec4ed8afacdc91df4a3cdfb6de0f7492d974d97" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ncar/{{ name|lower }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true # [win]
+
+requirements:
+  build:
+    - toolchain
+    - make
+    - netcdf-fortran 4.4.4
+    - gfortran >=6.0
```

Comment:
This is where we're currently stuck.

Reply:
@bartnijssen and @jhamman take a look at the [`scipy` recipe](https://github.com/conda-forge/scipy-feedstock/tree/master/recipe). It should have everything you need to get this one working.  I am away from my laptop, travelling to the USA for a conference, but I can give this a try once I get back.

PS: note that we are installing Fortran with the `yum_requirements.txt` file instead of using the compilers in the default channel.

Reply:
BTW using gfortran >=6 here, which I don't think we have either in conda-forge or in defaults, would require us to rebuild openblas, etc with that Fortran.

Reply:
@ocefpaf : This is indeed the problem. The SUMMA code itself uses fortran features that are part of the standard, but which were not fully implemented in gfortran 4.8.* which is the default gcc version in conda-forge (and I think for a lot of the standard linux distributions). This requires rebuilding some of the libraries (netcdf, blas) to be consistent, which is the challenge. I'll take a closer look at the scipy recipe, but perhaps waiting for the compiler upgrade before adding this to conda-forge may be easier.

Reply:
I double checked and `defaults` does have a modern Fortran we could use, but that would mean a deeper change in conda-forge (that must happen at sometime) to adopt those compilers.

It would require to re-build everything that depends on Fortran, starting from openblas.

TL;DR I don't think we are ready for this just yet. Conda-forge must move to conda-build 3, then adopt the modern compiler, then re-build the whole Fortran stack. It will take a while, sorry,

Ping: @mingwandroid who is maintaining that toolchain.

Reply:
Thanks for looking into this. This is precisely the feedback I needed (although not the answer I had hoped for :) ). I started putting this together thinking it was easy, but the compiler issue tripped us up. I will go a different route for now.

Reply:
> This is precisely the feedback I needed (although not the answer I had hoped for :) 

Stay tuned though. Ee do want to start using conda-build 3 and the compilers from the default channel ASAP.